korangar/src/world/entity/mod.rs

```diff
@@ -449,7 +452,7 @@ impl Common {
 
                 let array = last_step_position - next_step_position;
                 let array: &[isize; 2] = array.as_ref();
-                self.head_direction = match array {
+                self.direction = match array {
                     [0, 1] => 0,
```

Comment:
You can use the `Direction` enum here now! To remove these magic values is the reason why we added it!